Sources/PointFree/Newsletters.swift

```diff
@@ -1,13 +1,55 @@
 import ApplicativeRouter
+import Either
 import HttpPipeline
 import Foundation
 import Prelude
 import Tuple
 
 let expressUnsubscribeMiddleware =
-{ (conn: Conn<StatusLineOpen, T4<Database.User?, Database.User.Id, Database.EmailSetting.Newsletter, Prelude.Unit>>) -> IO<Conn<StatusLineOpen, Prelude.Unit>> in
+  unsubscribeMiddleware
```

Comment:
this one `unsubscribeMiddleware` gets reused to handle both situations of unsubscribing: either via a link or email reply.